main.tf

```diff
@@ -1,3 +1,29 @@
-/********************************************************************
-This file is used to implement the ROOT module.
-*********************************************************************/
+##############################################################################
+# Get Cloud Account ID
+##############################################################################
+
+data "ibm_iam_account_settings" "iam_account_settings" {
```

Comment:
we have documented that you need:
```
Account Management
   Editor role access
```
However I thought this data lookup required admin access (I could be wrong). Can we confirm?

Reply:
Editor access is adequate